pkg/csi/cinder/openstack/openstack.go

```diff
@@ -206,3 +214,13 @@ func GetOpenStackProvider() (IOpenStack, error) {
 
 	return OsInstance, nil
 }
+
+//GetMaxVolLimit returns max vol limit
+func GetMaxVolLimit() int64 {
+	if cfg.BlockStorage.NodeVolumeAttachLimit > 0 {
```

Comment:
do we need to check here if its greater than 26 ?
And also , is this configurable on cinder side also? if yes, then we need to get exact configured value for that node from cinder?

Reply:
@ramineni yes, will update the PR for greater than 26,
It is not currently configurable, work in progress in openstack